build.gradle

```diff
@@ -9,23 +9,14 @@
 
 buildscript {
     ext {
-        kotlin_version = '1.8.0'
+        kotlin_version = '1.9.10'
         customCSSFile = projectDir.toString() + "/dokka/fr-backstage-styles.css"
         customLogoFile = projectDir.toString() + "/dokka/logo-icon.svg"
         customTemplatesFolder = file(projectDir.toString() + "/dokka/templates")
     }
-    repositories {
```

Comment:
moved all the repositories to settings.xml . @spetrov we may need to test the maven release to make sure this change works